src/com/rcv/TieBreak.java

```diff
@@ -184,19 +186,21 @@ private String doPreviousRounds() {
     // round indexes from the previous round back to round 1
     for (int round = this.round - 1; round > 0; round--) {
       // map of tally to candidate IDs for round under consideration
-      SortedMap<BigDecimal, LinkedList<String>> tallyToCandidates = Tabulator.buildTallyToCandidates(
-        roundTallies.get(round),
-        candidatesInContention,
-        false
-      );
+      SortedMap<BigDecimal, LinkedList<String>> tallyToCandidates = Tabulator
+          .buildTallyToCandidates(
```

Comment:
nit: would prefer to put `Tabulator.buildTallyToCandidates(` together on this line

Reply:
Done.